### DISCUSSION TITLE
I can't get CTRLR to work with my Yamaha Reface CS

### DISCUSSION THREAD

**Discussion by deanhuff** [RomanKubiak/ctrlr#502] [Q&A] [answered]
Hello Community,

I am at your mercy.  I have 4 computers (iMac, Fedora workstation, Ubuntu laptop, Microsoft Surface).  On each of these computers I am able to browse to the Soundmondo website using chrome and read/write and monitor my CS.  On each of these computers, I have failed to get CTRLR running succesfully.

At this point, the only feature I am looking for is the ability to store and load my patches.

On my linux machines, I've attempted to use the latest 5.5.9 stable release.  Additionally on all of my machines, I've attempted to use the 5.6.0 pre-release.  Each fails with a different result.

I am attempting to use the Reface CS panels from here:  http://ctrlr.martintarenskeen.nl/
I have tried both of these versions:  

- [Yamaha Reface CS editor/librarian for Ctrlr](http://home.kpn.nl/m.tarenskeen/download/sysex/Ctrlr/refacecs-20210405.bpanelz.zip) (version 20210405)

- [Yamaha Reface CS editor/librarian for Ctrlr](http://home.kpn.nl/m.tarenskeen/download/sysex/Ctrlr/refacecs-20220908.bpanelz.zip) (version 20220908) 

On my mac, I can open the CTRLR interface but when I attempt to open the *.bpanelz file, no files are listed in the file chooser.  Also there is no option that I can see to specify a file type (extension) in the file chooser so I am stuck.

On the Windows surface laptop, I can open the CTRLR interface, load the *.bpanelz.  I see my device 'reface CS' in the midi menu and select it.  I see a green bar at the bottom of the CTRLR window briefly.  When I click get/push nothing happens in either the screen or sounds from the device.  When I move the sliders on the device, the changes are not shown in the CTRLR Panel.

On the 2 linux machines, When I open the CTRLR interface and select the .bpanelz file, the application immediately crashes with the following:
```
Ctrlr crash at: 16 Nov 2022 13:07:28
Stack trace:
./Ctrlr(+0x6506ee) [0x55c0656c26ee]
./Ctrlr(+0x58ae1b) [0x55c0655fce1b]
./Ctrlr(+0x63e151) [0x55c0656b0151]
/lib64/libc.so.6(+0x3ea30) [0x7f7308c3ea30]
./Ctrlr(+0x90581a) [0x55c06597781a]
./Ctrlr(+0x905bf8) [0x55c065977bf8]
./Ctrlr(+0xa45218) [0x55c065ab7218]
./Ctrlr(+0xa4e033) [0x55c065ac0033]
./Ctrlr(+0xa4ebf6) [0x55c065ac0bf6]
./Ctrlr(+0x94153f) [0x55c0659b353f]
./Ctrlr(+0x92f1b6) [0x55c0659a11b6]
./Ctrlr(+0x92f45f) [0x55c0659a145f]
./Ctrlr(+0x92f630) [0x55c0659a1630]
./Ctrlr(+0x59a902) [0x55c06560c902]
./Ctrlr(+0x6da29a) [0x55c06574c29a]
./Ctrlr(+0x6d922c) [0x55c06574b22c]
./Ctrlr(+0x6d9467) [0x55c06574b467]
./Ctrlr(+0x28d860) [0x55c0652ff860]
/lib64/libc.so.6(+0x29510) [0x7f7308c29510]
/lib64/libc.so.6(__libc_start_main+0x89) [0x7f7308c295c9]
./Ctrlr(+0x58ab1e) [0x55c0655fcb1e]
```
Any help would be greatly appreciated.

**Comment by deanhuff** [comment, marked as answer]
I have had success on my iMac.  The setting EDIT > PREFERENCES > UNCHECK "Use OS native dialog windows" allowed me to open the bpanelz file.

The next hurdle was on the OSX version it was reporting it could not connect to MIDI.  Per this issue:  https://github.com/RomanKubiak/ctrlr/issues/373  there is an OSX build located here:  https://github.com/jmechnich/ctrlr/releases/tag/5.6-f452347 that works.

Linux and windows continue to not work for me as documented above.

So I'm "partially" answered.  Should I mark this as answered?

-Dean

**Reply by damiensellier** [reply]
Hi Dean, glad you discovered the  **EDIT > PREFERENCES > UNCHECK "Use OS native dialog windows"** trick. 
It's required for all macOS version including BIGSUR and above.
Maybe someone will help you with Windows and Linux, I never use them.
Damien

**Reply by dnaldoog** [reply]
Hi Dean,

For Windows and Linux use version 5.3.201
For MacOS 5.3.198 is known to be the most stable.

**Reply by deanhuff** [reply]
@dnaldoog  thank you, yes 5.3.201 works for me on linux.  getting it working on a portable is of the most use for me.  thanks again for the help!
